templates/zerver/help/about-streams-and-topics.md

```diff
@@ -17,50 +17,50 @@ only visible to users who've been invited to subscribe to them. Users who
 are not members of a private stream cannot subscribe to the stream, and
 they also cannot read or send messages to the stream.
 
-Users are subscribed to specific streams in the organization by default, such as
-the [#announce](the-announce-stream) stream. Users can easily
-[view messages](/help/view-messages-from-a-stream) from a specific stream; in
-addition, they can [browse](/help/browse-and-join-streams#browse-streams) their
-stream subscriptions using the Zulip stream browser.
+Users are subscribed to specific streams in the organization by default,
+such as the [#announce](the-announce-stream) stream. Users can easily
+[view messages](/help/view-messages-from-a-stream) from a specific stream;
+in addition, they can
+[browse](/help/browse-and-subscribe-to-streams#browse-streams) their stream
+subscriptions using the Zulip stream browser.
 
 If they wish to read messages from a stream that they're not subscribed to,
-users can choose to [join](/help/browse-and-join-streams#subscribing-to-streams)
-a stream. Similarly, if they are not interested in the topics being discussed in
-a stream, users can choose to [unsubscribe](/help/unsubscribe-from-a-stream) from a
-stream.
-
-Users can also customize their stream settings; they can:
+users can choose to
+[join](/help/browse-and-subscribe-to-streams#subscribing-to-streams) a
+stream. Similarly, if they are not interested in the topics being discussed
+in a stream, users can choose to
+[unsubscribe](/help/unsubscribe-from-a-stream) from a stream.  Users can
+also customize their stream settings; they can:
 
 * [pin a stream](/help/pin-a-stream)
 * [change the color of streams](/help/change-the-color-of-a-stream)
 * [enable desktop notifications for stream](/help/configure-desktop-notifications)
 * [muting notification for streams](/help/mute-a-stream)
 
 If enabled by the organization administrators, users can
-[create](/help/create-a-stream) streams and [invite](/help/add-or-invite-someone-to-a-stream)
-other users to a stream.
+[create](/help/create-a-stream) streams and
+[invite](/help/add-or-invite-someone-to-a-stream) other users to a stream.
 
-Only organization administrators can modify a stream, like:
+Only organization administrators can modify a stream, such as:
 
 * [renaming a stream](/help/rename-a-stream)
 * [deleting a stream](/help/delete-a-stream)
-* changing the description of a stream
+* [changing the description of a stream](/help/change-the-stream-description)
 * [removing users from a stream](/help/remove-someone-from-a-stream)
-* [changing the accessibility of a stream](/help/change-who-can-join-a-stream)
+* [changing the accessibility of a stream](/help/change-the-accessibility-of-a-stream)
```

Comment:
Wondering if it should be consistent with `changing the description of a stream`